How_to_add_a_new_data_type.md

```diff
@@ -380,19 +382,19 @@ Standard input keywords:
 
 > **Landing page / Narrative Visualization Widget**
 >
-> Your data type should have some minimal visualization in both the Narrative interface (so that clicking on the data object in the left-hand data list or dragging and dropping on to the narrative cells opens the viewer) and in the functional-site to show on the landing page for your data. Visualizations should be primarily keyed off of a single typed object defined and registered with the Workspace.
+> Your data type should have some minimal visualization in both the Narrative interface (so that clicking on the data object in the left-hand data list or dragging and dropping onto the Narrative panel opens the viewer) and in the functional site to show on the landing page for your data. Visualizations should be primarily keyed off of a single typed object defined and registered with the Workspace.
```

Comment:
Yes--it becomes a Narrative cell when you drag it onto the Narrative panel. "Narrative panel" (or "main Narrative panel") was the terminology we agreed on for this section of the NI. You're not dragging the object onto a cell.
I don't feel strongly about this, though, so if you object to this change, I'm not going to push it.